### PR TITLE
Feature/13030 poll fss for batch status

### PR DESCRIPTION
**Pull request by chaital14503**
Code changes for removing spinner after batch status is "Committed", and download button implementation. 
Kindly review the changes.

### CODE REVIEW

**Review by crdandoukho — CHANGES_REQUESTED** [review]

File: src/app/features/exchange-set/ess-download-exchangeset/ess-download-exchangeset.component.ts [hunk 1]
```diff
@@ -1,36 +1,104 @@
...
+        .loginPopup(this.fssSilentTokenRequest)
+        .then(response => {
+          this.batchStatusAPI();
+        })
+    })
+  }
+
+  batchStatusAPI() {
+    this.fileShareApiService.getBatchStatus(this.batchId).subscribe((response) => {
+      if (response.status == "Committed") {
```
[the remark this answers is not in the document]
Comment (chaital14503) [reply src/app/features/exchange-set/ess-download-exchangeset/ess-download-exchangeset.component.ts:72]: Added another condition for batchstatus Incomplete or CommitInProgress, and else part for Rolledback and Failed. UTs updated accordingly. Thanks.

**Review by crdandoukho — APPROVED** [review]
Looks good

**Review by rvijayavel — APPROVED** [review]
Looks good.

**Review by crdandoukho — APPROVED** [review]
Looks good

**Pull request merged**